apps/src/netsim/NetSimLogPanel.js

```diff
@@ -308,6 +310,52 @@ NetSimLogPacket.prototype.getRoot = function () {
 };
 
 /**
+ * Beneath the given root element, adjust widths of packet header columns
+ * and fields to match the level's configured packet format.
+ * @param {jQuery} rootElement
+ */
+NetSimLogPanel.adjustHeaderColumnWidths = function (rootElement) {
+  var level = netsimGlobals.getLevelConfig();
+  var encoder = new Packet.Encoder(
+      level.addressFormat,
+      level.packetCountBitWidth,
+      level.clientInitialPacketHeader);
+  var addressBitWidth = encoder.getFieldBitWidth(
+      Packet.HeaderType.TO_ADDRESS);
+  var packetInfoBitWidth = encoder.getFieldBitWidth(
+      Packet.HeaderType.PACKET_COUNT);
+
+  // Adjust width of address columns
+  // For columns, 50px is sufficient for 4 bits
+  var pxPerBit = 50 / 4;
+  var addressColumnWidthInPx = pxPerBit * addressBitWidth;
+
+  // Adjust width of address columns
+  rootElement.find('td.toAddress, th.toAddress, td.fromAddress, th.fromAddress')
+      .css('width', addressColumnWidthInPx + 'px');
+
+
+  // Adjust width of address input fields
+  // For inputs, 3em is sufficient for 4 bits
+  var emsPerBit = 3 / 4;
+  var addressFieldWidthInEms = emsPerBit * addressBitWidth;
+  rootElement.find('td.toAddress input, td.fromAddress input')
+      .css('width', addressFieldWidthInEms + 'em');
+
+
+  // Adjust width of packet info column
+  // Packet info column uses two fields and an extra 21px for " of "
+  var packetInfoColumnWidthInPx = (2 * pxPerBit * packetInfoBitWidth) + 21;
```

Comment:
Actually, this looks pretty good across all the browsers I tested.  Given that the ed team is likely to pick one or two address formats and stick with them, I'm inclined to leave this until something comes up.